internal/zinc-compile-core/src/test/scala/sbt/internal/inc/javac/JavaCompilerSpec.scala

```diff
@@ -95,7 +95,7 @@ class JavaCompilerSpec extends UnitSpec {
         val (result, _) = compile(local, Seq(input), Seq("-d", out.getAbsolutePath))
         result shouldBe true
         val clazzz = new URLClassLoader(Array(out.toURI.toURL)).loadClass("hasstaticfinal")
-        ClassToAPI(Seq(clazzz))
+        ClassToAPI(Seq(clazzz)).sortBy(_.definitionType.hashCode)
```

Comment:
@zehkae Thanks for the explanation!
  
So I have two questions:
  
1. Could this issue be fixed in Contraband?
2. When you mention `leftAPI` and `rightAPI`, do you mean that their order alter the behaviour of the API change detection algorithm?
  
To me, this change seems to be necessary only for the testing suite. I'm slightly reluctant of sorting this only for testing purposes because order should not be a factor that we should take into account if it really doesn't alter semantics :smile:.